compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/statements/BLangBlockStmt.java

```diff
@@ -39,6 +39,8 @@
 
     public FailureBreakMode failureBreakMode = FailureBreakMode.NOT_BREAKABLE;
 
+    public boolean isLetExprStmt = false;
```

Comment:
Since it is a block statement, we can remove the `stmt` potfix in this variable and keep it as `isLetExpr` only.